templates/default/eslintignore

```diff
@@ -0,0 +1,2 @@
+**/*.test.js
```

Comment:
revert this please, test files should have code linting consistent with functional files